code/modules/antagonists/bloodsucker/bloodsucker_life.dm

```diff
@@ -247,10 +257,11 @@
 		if(power.active && !power.can_use_in_torpor)
 			power.DeactivatePower()
 
-
 /datum/antagonist/bloodsucker/proc/Torpor_End()
 	owner.current.stat = SOFT_CRIT
-	owner.current.cure_fakedeath("bloodsucker") // Come after SOFT_CRIT or else it fails
+	owner.current.remove_status_effect(STATUS_EFFECT_UNCONSCIOUS)
+	owner.current.cure_fakedeath("bloodsucker")
```

Comment:
It doesn't even use fakedeath() to start with anymore.

Reply:
Removed it, tested and it didnt break anything